### ISSUE TITLE
IntrospectionProcessor undefined offset

### ISSUE THREAD

**Issue by mtibben** [Seldaek/monolog#624]
While logging in a Symfony application using the Monolog 1.16.0 and the IntrospectionProcessor, the following error occurred.

```
Fatal error: Uncaught exception 'Symfony\Component\Debug\Exception\ContextErrorException' with message 'Notice: Undefined offset: 2' in /myproject/vendor/monolog/monolog/src/Monolog/Processor/IntrospectionProcessor.php:64
Stack trace:
#0 /myproject/vendor/monolog/monolog/src/Monolog/Processor/IntrospectionProcessor.php(64): Symfony\Component\Debug\ErrorHandler->handleError(8, 'Undefined offse...', '/myproject/vendo...', 64, Array)
#1 [internal function]: Monolog\Processor\IntrospectionProcessor->__invoke(Array)
#2 /myproject/vendor/monolog/monolog/src/Monolog/Logger.php(286): call_user_func(Object(Monolog\Processor\IntrospectionProcessor), Array)
#3 /myproject/vendor/monolog/monolog/src/Monolog/Logger.php(481): Monolog\Logger->addRecord(100, 'debug', Array)
#4 /myproject/scripts/testLogging.php(8): Monolog\Logger->debug('debug')
#5 {main}
  thrown in /myproject/vendor/monolog/monolog/src/Monolog/Processor/IntrospectionProcessor.php on line 64
```

It looks like an issue with recent changes to IntrospectionProcessor

**Comment by frederikbosch** [comment]
Just fixed this with #625.

**Comment by mtibben** [comment]
Great thanks guys
